package.json

```diff
@@ -8,37 +8,28 @@
   "module": "es/index.js",
   "jsnext:main": "es/index.js",
   "scripts": {
-    "build:types": "cpy src/index.js.flow lib",
+    "build:flow": "cpy src/index.js.flow lib & cpy src/index.js.flow es",
     "build:commonjs": "cross-env BABEL_ENV=commonjs babel src --out-dir lib",
     "build:es": "cross-env BABEL_ENV=es babel src --out-dir es",
-    "build:umd": "cross-env BABEL_ENV=rollup NODE_ENV=development rollup -c -o dist/react-powerplug.js",
-    "build:umd:min": "cross-env BABEL_ENV=rollup NODE_ENV=production rollup -c -o dist/react-powerplug.min.js",
-    "build:all": "npm run build:commonjs && npm run build:es && npm run build:umd && npm run build:umd:min && npm run build:types",
+    "build:umd":
+      "cross-env BABEL_ENV=rollup NODE_ENV=development rollup -c -o dist/react-powerplug.js",
+    "build:umd:min":
+      "cross-env BABEL_ENV=rollup NODE_ENV=production rollup -c -o dist/react-powerplug.min.js",
+    "build:all":
+      "npm run build:commonjs && npm run build:es && npm run build:umd && npm run build:umd:min && npm run build:flow",
     "build": "npm run clean && npm run build:all",
     "clean": "rimraf lib dist es",
-    "typecheck": "flow check && [[ $(flow check) == 'Found 0 errors' ]]",
+    "typecheck:flow": "flow check",
     "lint": "eslint src",
     "test": "jest",
-    "posttest": "npm run typecheck",
-    "prepublishOnly": "npm run lint && npm run clean && npm run build && npm run typecheck"
+    "prepublishOnly": "npm run lint && npm run clean && npm run build"
   },
   "repository": {
     "type": "git",
     "url": "https://github.com/renatorib/react-powerplug.git"
   },
-  "files": [
-    "dist",
-    "lib",
-    "src",
-    "es"
-  ],
-  "keywords": [
-    "react",
-    "reactjs",
-    "components",
-    "dumb",
-    "state"
-  ],
+  "files": ["dist", "lib", "src", "es"],
+  "keywords": ["react", "reactjs", "components", "dumb", "state"],
```

Comment:
Do you have automatic formatter for this?

Reply:
I use prettier in my editor 😅

Reply:
I need to create a .prettierrc file

Reply:
+1